### DISCUSSION TITLE
Add permission for min scale and max scale

### DISCUSSION THREAD

**Discussion by NoltoxGit** [Zrips/CMI#9069] [Enhancement]
Hello,
I'd like to give players permission to do `/scale`. But I don't want them to be able to scale 16 or 0.1.
Would it be possible to add permissions for scale (a bit like flyspeed, for example)?

## Examples of permissions:
`cmi.command.scale.min.1`: A player won't be able to scale below 1.
`cmi.command.scale.max.3`: A player cannot scale above 3.

**Comment by 7hatGuyAJ** [comment]
I can confirm this is a much needed permission node in order to give players access to scale command because anything below scale 0.4 causes the player to block clip giving them the ability to X-ray caves, trial chambers, etc.

**Reply by SinderellaDeVita** [reply]
Much needed permission node. Players would really enjoy scaling. But as mentioned above it block clips 0.4 and under.

**Comment by BAD7777** [comment]
up
